src/app/component/auth/components/sign-up/sign-up.component.ts

```diff
@@ -88,6 +89,7 @@ export class SignUpComponent implements OnInit {
           break;
         case 'email':
           this.emailErrorMessageBackEnd = error.message;
+          this.isEmailInvalid = this.emailErrorMessageBackEnd === 'The email is invalid' ? true : false;
```

Comment:
Is it necessary to write "this.emailErrorMessageBackEnd === 'The email is invalid' ? true : false"? Try to rewrite it just as "this.emailErrorMessageBackEnd === 'The email is invalid'".

Reply:
thanks, fixed